src/uci.cpp

```diff
@@ -40,12 +40,12 @@ extern vector<string> setup_bench(const Position&, istream&);
 
 namespace {
 
-  // FEN string of the initial position, normal chess
+  // A FEN string for the initial position in standard chess.
   const char* StartFEN = "rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1";
 
 
-  // position() is called when engine receives the "position" UCI command.
-  // The function sets up the position described in the given FEN string ("fen")
+  // position() is called when the engine receives the "position" UCI command.
+  // The function sets up a position described in the given FEN string ("fen")
```

Comment:
This change is incorrect, the definite article "the" is appropriate for *the* position described in *the* given FEN string